### PR TITLE
Update core.py

### PR DESCRIPTION
**Pull request by zleite**
### What does this PR do?
Trivial change.
Add a grain with a list of networks the minion is connected to.
### What issues does this PR fix or reference?
Fixes:
Networks / netmasks are core, critical information about the minion.
### Previous Behavior
No "networks" grain.
### New Behavior
"networks" grain present.

### CODE REVIEW

**Comment by MKLeb** [comment]
Hi @zleite!  Would you mind writing a test for this change and adding a changelog?  We require these before merging.

File: salt/grains/core.py [hunk 1]
```diff
@@ -2559,6 +2559,14 @@ def ip6_interfaces():
     return {"ip6_interfaces": ret}
 
 
+def networks():
+    """
+    Provides a list of connected networks
```
Comment (MKLeb) [inline salt/grains/core.py:2564]:
```suggestion
    Provides a list of connected networks
```

Comment (MKLeb) [reply salt/grains/core.py:2564]: I think this should fix the pre-commit and lint issues

**Comment by dwoz** [comment]
Closing this due to inactivity. Anyone should feel free to re-open it if they want to see it through to the end in one release cycle.

**Pull request closed without merging**